akka-projection-dynamodb/src/main/scala/akka/projection/dynamodb/internal/DynamoDBProjectionImpl.scala

```diff
@@ -909,6 +676,76 @@ private[projection] object DynamoDBProjectionImpl {
     }
   }
 
+  private def replayIfPossible[Offset, Envelope](
+      offsetStore: DynamoDBOffsetStore,
+      sourceProvider: SourceProvider[Offset, Envelope],
+      originalEnvelope: Envelope)(accepted: EventEnvelope[Any] => Future[Done])(
+      implicit ec: ExecutionContext,
+      system: ActorSystem[_]): Future[Boolean] = {
+    val logPrefix = offsetStore.logPrefix
+    originalEnvelope match {
+      case originalEventEnvelope: EventEnvelope[Any @unchecked] if originalEventEnvelope.sequenceNr > 1 =>
+        val underlyingProvider = sourceProvider match {
+          case adapted: JavaToScalaBySliceSourceProviderAdapter[_, _] => adapted.delegate
+          case provider                                               => provider
+        }
+        underlyingProvider match {
+          case provider: LoadEventsByPersistenceIdSourceProvider[Any @unchecked]
+              if offsetStore.settings.replayOnRejectedSequenceNumbers =>
+            val persistenceId = originalEventEnvelope.persistenceId
+            offsetStore.storedSeqNr(persistenceId).flatMap { storedSeqNr =>
+              val fromSeqNr = storedSeqNr + 1
+              val toSeqNr = originalEventEnvelope.sequenceNr
+              logReplayRejected(logPrefix, persistenceId, fromSeqNr, toSeqNr)
+              provider.currentEventsByPersistenceId(persistenceId, fromSeqNr, toSeqNr) match {
+                case Some(querySource) =>
+                  querySource
+                    .mapAsync(1) { envelope =>
+                      import DynamoDBOffsetStore.Validation._
+                      offsetStore
+                        .validate(envelope)
+                        .flatMap {
+                          case Accepted =>
+                            accepted(envelope)
```

Comment:
I looked at diffs of the the code that were in the handlers and the only difference was how the `Accepted` case was handled.

Reply:
Yeah, seemed that would be the case.